template/script.sh.erb

```diff
@@ -3,6 +3,13 @@
 <%-
   require 'pathname'
 
+  # Check if we can enable JupyterLab extension
+  extension_enabled = false
+  version = context.version.split('/').last
+  if context.jupyterlab_switch == "1" and context.jupyterlab_use_extension == "1" and version.split('.').first.to_i > 0
```

Comment:
Actually, now that we're using version here too, let's get a function for version itself and call it in the if expression here as

```ruby 
# 'unknown' will case to 0
version.to_i  > 0
```
With the method being this. We can then use the same method below to figure out `jupyterlab_dir`.
```ruby
def version
  match = context.version.match('app_jupyter/(?<version>\d\.\d)')
  if !match.nil? && match.named_captures.key?('version')
    match[:version]
  else
    'unknown'
   end
end
```

Reply:
I simplify the logic to enable extension and use jupyterlab_dir in 6dc7d2b8c7ed25df2db8247494d0bebd6c5164e3. Just realize that I need to add the variable `use_extension` in classroom form. Should I update osc-ood-config before doing this pull request?